src/services/InitService.js

```diff
@@ -0,0 +1,55 @@
+import * as dotenv from 'dotenv';
+import ContextBuilder from '../utilities/ContextBuilder';
+import LoggingService from './LoggingService';
+import Api from '../api/Api';
+
+dotenv.config();
+
+const init = async (request) => {
+  const totalValue = 30 + parseInt(request.message.order.items[0].price.value, 10);
+  const data = {
+    order: {
+      provider: {
+        id: request.message.order.provider.id,
+      },
+      items: request.message.order.items,
+      billing: request.message.order.billing,
+      quote: {
+        price: {
+          currency: request.message.order.items[0].price.currency,
+          value: totalValue,
+        },
+        breakup: [
+          {
+            title: 'Fare',
+            price: request.message.order.items[0].price,
+          },
+          {
+            title: 'Tax',
+            price: {
+              currency: 'INR',
+              value: '30',
+            },
+          },
+        ],
+      },
+      payment: request.message.order.payment,
+    },
+  };
+  const logger = LoggingService.getLogger('InitService');
+  logger.debug(JSON.stringify(request));
+  const response = {
+    context: ContextBuilder.getContextWithContext(request.context),
+    message: data,
+  };
+  // eslint-disable-next-line no-promise-executor-return
+  await new Promise((resolve) => setTimeout(resolve, 3000)); // delayed result
```

Comment:
No need to delay here. I added delay for search to look real

Reply:
It has been removed.